SU-SOE/soe_basic
# D8CORE-3154: fixing the heading spacing on people

# READY FOR REVIEW

# Summary
- Fixing the SOE Person heading spacing

# Review By (Date)
- 1/2

# Urgency
- low

# Steps to Test

1. Pull in this change
2. View the all person page. View the filtered pages.
3. View Events.
4. Verify all the spacing around the h1 is consistent on the News, Events, Event Series, Person pages. It has to work on the Person filtered pages as well. 

# Affected Projects or Products
- SOE subtheme

# Associated Issues and/or People
- D8CORE-3154

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
